setup.cfg

```diff
@@ -41,6 +41,8 @@ install_requires =
     pyyaml
     importlib-resources; python_version < "3.9.0"
     dataclasses; python_version < "3.7.0"
+    # used for TypeAlias, can be removed once Python 3.10 is required
+    typing-extensions;python_version < '3.10'
```

Comment:
```suggestion
    typing-extensions; python_version < '3.10'
```

Reply:
I removed the space on purpose to match the format in the example: https://www.python.org/dev/peps/pep-0508/#environment-markers

But I can reverse that.